src/transformers/models/auto/tokenization_auto.py

```diff
@@ -24,8 +24,14 @@
 from ...dynamic_module_utils import get_class_from_dynamic_module
 from ...tokenization_utils import PreTrainedTokenizer
 from ...tokenization_utils_base import TOKENIZER_CONFIG_FILE
-from ...tokenization_utils_fast import PreTrainedTokenizerFast
 from ...utils import cached_file, extract_commit_hash, is_sentencepiece_available, is_tokenizers_available, logging
+
+
+if is_tokenizers_available():
+    from ...tokenization_utils_fast import PreTrainedTokenizerFast
+else:
+    from ...utils.dummy_tokenizers_objects import PreTrainedTokenizerFast
+
```

Comment:
No there is no need for a dummy AutTokenizer (especially since AutoTokenizer has no hard-dependency on the tokenziers library). In the else statement, `PreTrainedTokenizerFast` should just be set to `None` and it will work.